fmriprep/workflows/bold/base.py

```diff
@@ -940,26 +946,37 @@ def init_func_preproc_wf(
 *Grayordinates* files [@hcppipelines], which combine surface-sampled
 data and volume-sampled data, were also generated.
 """
+            cifti_volume = "MNI152NLin6Asym" if 'fsLR' in cifti_spaces else "MNI152NLin2009cAsym"
```

Comment:
I thought they used an FSL-packaged (and slightly weird) version of the template. Do we need to account for that?

Reply:
we are using the same [labelled MNI template](https://github.com/Washington-University/HCPpipelines/blob/master/global/templates/91282_Greyordinates/Atlas_ROIs.2.nii.gz) the HCP folks use to generate the CIFTIs, which should be in same space as the template - what do you mean by slightly weird?

Reply:
I might be misremembering. During the TemplateFlow changeover we switched from using FSL's packaged template (MNI152NLin2009cAsym) to MNI's, and there were some differences. I thought using FSL's distribution was something we'd copied from HCP. @oesteban will probably know better.

Reply:
> I thought they used an FSL-packaged (and slightly weird) version of the template. Do we need to account for that?
The "MNI" distributed with FSL is an unofficial version created by A. Janke with the same data and methodology of the official `MNI152NLin6Sym` atlas. We called this variant `MNI152NLin6Asym` in TemplateFlow, although it is not an official MNI template.

Then, the HCP folks resampled the 1mm^3 grid to different resolutions that seem useful to them. We originally placed these files under the `fsLR` ID in TemplateFlow. @mgxd refactored TemplateFlow and moved them to their (more proper) place in `MNI152NLin6Asym`.

During the TemplateFlow changeover, we did not switch from the official, latest MNI template (`MNI152NLin2009cAsym`) to MNI's (I guess you were referring to `MNI152NLin6Asym`). What really happened is that some submodules (and particularly, ICA-AROMA) stopped using the `MNI152NLin2009cAsym` template. After the changeover, if you set `--ica-aroma` then the `MNI152NLin6Asym` is added to the output-spaces list because ICA-AROMA was originally trained on that space.

Hopefully, this clarifies the spotty connections between TemplateFlow, the "MNI" template(s) and this PR.